Tests/SwiftParserTest/translated/OperatorDeclTests.swift

```diff
@@ -480,6 +480,41 @@ final class OperatorDeclTests: XCTestCase {
     )
   }
 
+  
+  func testOperatorDecl124() {
+    assertParse(
+    """
+    prefix 1️⃣mutating operator ?? : UndeclaredPrecedenceGroup
+    """,
+    diagnostics: [
+        DiagnosticSpec(message: "unexpected code 'mutating' in operator declaration")
+      ]
+    )
+  }
+  
+  func testOperatorDecl125() {
+    assertParse(
+    """
+    prefix 1️⃣postfix operator ** : PrecedenceGroup
+    """,
+    diagnostics: [
+        DiagnosticSpec(message: "unexpected code 'postfix' in operator declaration")
```

Comment:
The old parser generates *'postfix' contradicts previous modifier 'prefix'* here. If my changes work, I could optimize the diagnostics message here later.